testsuite/tests/backtrace/backtrace_bounds_exn.ml

```diff
@@ -0,0 +1,20 @@
+(* TEST
+   flags = "-g"
+   ocamlrunparam += ",b=1"
+*)
+
+(* bad bounds exception *)
+
+let xs = [| 0; 1; 2 |]
+
+let bad_bound_fn x =
```

Comment:
I think you should add `[@inline never]` to this function, to ensure the test remains robust.
Please also add a comment with the PR number.